.travis.yml

```diff
@@ -0,0 +1,25 @@
+{
+  language: objective-c,
+  before_install: [
+    "brew update"
+  ],
+  install: [
+    "mkdir -p $(brew --repo)/Library/Taps/travis",
+    "ln -s $PWD $(brew --repo)/Library/Taps/travis/homebrew-moira",
+    "brew tap --repair"
+  ],
+  env: PACKAGE=moira,
+  script: [
+    brew audit $PACKAGE,
+    brew install -v $PACKAGE,
+    brew test $PACKAGE
+  ],
+  matrix:
+  	include:
+    - os: osx
```

Comment:
should include `env: OSX=n.n`

Reply:
I don't see that in their documentation; is that actually necessary?

Reply:
The way your build matrix is currently written, you would be building on OS X 10.11 with Xcode 7.3.1 (the latest for OS X 10.11 is 8) and OS X 10.12 with Xcode 8.3 and 9.1

So I think if you want to stick to this fashion, you want to add additional `osx_image_ keys such as `xcode6.4`, the only value available tha tprovides OS X 10.10, and all the way to latest.

Reply:
Yes, that's the ticket, I think. I'm wondering if the env is just so you have a log of which version the build ran on? I don't see anything in Travis CI that will tell me that